### ISSUE TITLE
Please consider to discontinue the practice of including a version in .pc file names

### ISSUE THREAD

**Issue by yurivict** [openbabel/openbabel#2216]
It used to be openbabel-2.0.pc, and now it is openbabel-3.pc

This practice just forces all projects to update their pkg-config arguments, whether this is needed or not. In case this isn't needed this is just an unnecessary disruption. In case there are changes that break this project the project still needs to be fixed. Encoding the version into .pc file doesn't accomplish anything.

**Comment by baoilleach** [comment]
You may be correct. It's for exactly this reason that we had an extensive beta period at the release of 3.0 to solicit feedback like this. Changing it now again would be disruptive. When Open Babel 4.0 is being prepared, please raise this issue.

**Comment by yurivict** [comment]
> Changing it now again would be disruptive.

Alternatively, you can add a copy of the .pc file with the name ```openbabel.pc``` so that both would work.

**Comment by baoilleach** [comment]
If you can figure out how to get the installer to a create soft link from openbabel-3.pc to openbabel.pc, we'll accept a PR. I don't know much about the conventions for pkg-config - are we sure that the correct convention is to leave out the major library version?

**Comment by yurivict** [comment]
> are we sure that the correct convention is to leave out the major library version?

Most projects don't include the major library version in their .pc files.

Imagine that there are 100 other projects that depend on OpenBabel-3. When you release OpenBabel-4, if .pc file has 3 in it, each of 100 projects needs to be manually patched just to switch the pkg-config argument. Most projects' maintainers aren't readily available to do this, and some projects are unmaintained.

**Comment by ghutchis** [comment]
I released Open Babel 2.0 in 2005, and v3.0 was released in 2019. That's 14 years of API compatibility.

One reason I *intentionally* added the version number to the pkg-config file was to signal a strong backwards-incompatible change.

One should not just make a simple patch to switch a pkg-config arrangement. My guess is that whenever 4.0 comes out, the 100 projects (or whatever) will similarly need to make updates - and possibly significant ones.

I don't have a problem with switching to `openbabel.pc` but frankly you, or any other package maintainer shouldn't just assume that projects will be compatible with another major version change by tinkering with the `pkg-config` or cmake files.
